modules/api-compatibility-exceptions.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * rest_api/understanding-compatibility-guidelines.adoc
+
+[id="api-compatibility-exceptions_{context}"]
+= API compatibility exceptions
```

Comment:
can we add the following exception:

*API availability per product installation topology*
The OpenShift distribution will continue to evolve its supported installation topology, and not all APIs in one install topology will necessarily be included in another.  For example, certain topologies may restrict read/write access to particular APIs if they are in conflict with the product installation topology or not include a particular API at all if not pertinent to that topology.  APIs that exist in a given topology will be supported in accordance with the compatibility tiers defined above.

Reply:
adding in new commit suggestion